test/algorithms/area/area_geo.cpp

```diff
@@ -81,20 +81,20 @@ void test_geo_strategies()
     CT err = 0.0000001;
 
     CT area_default = bg::area(geometry_geo);
-    BOOST_CHECK_CLOSE(area_default, 63316309346280.18, err);
+    BOOST_CHECK_CLOSE(area_default, 63316423532570.688, err);
     area = bg::area(geometry_geo, geographic_default);
-    BOOST_CHECK_CLOSE(area, 63316309346280.18, err);
+    BOOST_CHECK_CLOSE(area, 63316423532570.688, err);
 
     CT area_less_accurate = bg::area(geometry_geo, geographic_andoyer1);
-    BOOST_CHECK_CLOSE(area, 63316309346280.18, err);
+    BOOST_CHECK_CLOSE(area, 63316423532570.688, err);
     area = bg::area(geometry_geo, geographic_andoyer2);
-    BOOST_CHECK_CLOSE(area, 63316309224306.5, err);
+    BOOST_CHECK_CLOSE(area, 63316423410597.016, err);
     area = bg::area(geometry_geo, geographic_andoyer3);
-    BOOST_CHECK_CLOSE(area, 63316309224411.195, err);
+    BOOST_CHECK_CLOSE(area, 63316423410701.703, err);
     area = bg::area(geometry_geo, geographic_andoyer4);
-    BOOST_CHECK_CLOSE(area, 63316309224411.094, err);
+    BOOST_CHECK_CLOSE(area, 63316423410701.602, err);
     area = bg::area(geometry_geo, geographic_andoyer5);
-    BOOST_CHECK_CLOSE(area, 63316309224411.094, err);
+    BOOST_CHECK_CLOSE(area, 63316423410701.602, err);
```

Comment:
Looks cool.
I'm just wondering, how do we know it's better?

Reply:
it is with respect to geographiclib that gives `63316536351834.289`

I added also comparisons with geographiclib in `area_sph_geo.cpp` tests showing that BG after this PR is more accurate than before.